x-pack/solutions/security/plugins/security_solution_serverless/public/navigation/side_navigation.ts

```diff
@@ -29,6 +29,15 @@ export const initSideNavigation = async (services: Services) => {
         const footerGroup: GroupDefinition | undefined = draft.footer?.find(
           ({ type }) => type === 'navGroup'
         ) as GroupDefinition;
+        // Adds children for Stack Management panel
+        const management = footerGroup?.children.find((child) => child.link === 'management');
+        if (management) {
+          management.renderAs = 'panelOpener';
+          management.id = 'stack_management';
+          management.spaceBefore = null;
+          management.children = stackManagementLinks;
+          delete management.link; // no landing page for stack management
```

Comment:
This creates the user experience of has an icon of an arrow pointing to the right, which opens a side panel with Stack Management sub-navigation.